backend.native/llvm.list

```diff
@@ -0,0 +1 @@
+_Java_llvm_*
```

Comment:
Considering that `-exported_symbols_list` doesn't fail when wildcard doesn't match, it may make sense to omit package here (e.g. use `_Java_*`) to ensure this fix keeps working if we change the package name.